pandas/tseries/tests/test_resample.py

```diff
@@ -2931,6 +2931,21 @@ def test_consistency_with_window(self):
         self.assertEqual(result.index.nlevels, 2)
         tm.assert_index_equal(result.index.levels[0], expected)
 
+    def test_median_duplicate_columns(self):
+        # GH 14233
+
+        df = pd.DataFrame(np.array([[i + j for i in range(20)]
```

Comment:
Only difference is using `np.random.randn()` versus the sequence I put in there, which makes it easier to debug.  I can put the `randn()` if you want.